src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -413,6 +415,24 @@ public function iShouldSeeProductWithDiscountedUnitPriceInMyCart(string $product
         throw new \InvalidArgumentException(sprintf('The product %s does not exist', $productName));
     }
 
+    /**
+     * @Then /^the product "([^"]+)" should have total price ("[^"]+") in the cart$/
+     */
+    public function theProductShouldHaveTotalPriceInTheCart(string $productName, int $totalPrice): void
```

Comment:
If we use a transformer, we could just call `$product->getName()` in line 426 and remove exception from the end of the function